misc/ballerinalang-data-mapper/src/main/java/org/ballerinalang/datamapper/AIDataMapperCodeActionUtil.java

```diff
@@ -139,6 +139,12 @@ public static AIDataMapperCodeActionUtil getInstance() {
         }
 
         List<DiagnosticProperty<?>> props = diagnostic.properties();
+
+        //To check if the left is a symbol (Fix issue 30379)
```

Comment:
Do we need to mention the issue number here? We can have a test case instead.

Reply:
Removed the issue numb and added a test case.